modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -19,6 +19,11 @@ const defaultProps = {
   data: null,
   loader: Tiles3DLoader,
 
+  pickable: {type: 'boolean', value: false},
+  pickFeatures: {type: 'boolean', value: false},
+  autoHighlight: {type: 'boolean', value: false},
+  highlightColor: {type: 'array', value: [0, 0, 255, 150]},
```

Comment:
Please do not redefine existing props from the base Layer. This layer should behave consistently with the other layers.